client/js/global.js

```diff
@@ -21,7 +21,7 @@ function fmOverlay() {
   const $overlay = $('#fm-overlay');
   const $overlayWrapper = $overlay.children('.fm-overlay-wrapper');
   const $overlayFade = $overlay.children('.fm-overlay-fade');
-  const $overlayImage = $overlay.find('img', '.fm-image');
+  const $overlayImage = $overlay.find('img', '.fm-overlay-content.image');
```

Comment:
Changing `.fm-image` to `.fm-overlay-content.image` will ensure that only images inside of the overlay-image will be targeted and all images inside of a rich text overlay will be left alone.